docs/main/user-guides/replicated-storage-user-guide/advanced-operations/monitoring.md

```diff
@@ -101,4 +101,109 @@ Upon successful integration of the exporter with the Prometheus stack, the metri
+|replica_read_latency_us | Gauge | `name`=<replica_uuid>, `pv_name`=<pv_name>, `node`=<replica_node> | Integer | Total read latency for all IOs on replica in usec. |
+|replica_write_latency_us | Gauge | `name`=<replica_uuid>, `pv_name`=<pv_name>, `node`=<replica_node> | Integer | Total write latency for all IOs on replica in usec. |
+
+### Target/Volume IoStat Counters
+
+| Metric Name | Metric Type | Labels/Tags | Metric Unit | Description |
+| :--- | :--- | :--- | :--- | :--- |
+|volume_num_read_ops | Gauge | `pv_name`=<pv_name> | Integer | Number of read operations through vol target |
+|volume_bytes_read | Gauge | `pv_name`=<pv_name> | Integer | Total bytes read through vol target |
+|volume_num_write_ops | Gauge | `pv_name`=<pv_name> | Integer | Number of write operations through vol target |
+|volume_bytes_written | Gauge | `pv_name`=<pv_name> | Integer | Total bytes written through vol target |
+|volume_read_latency_us | Gauge | `pv_name`=<pv_name> | Integer | Total read latency for all IOs through vol target in usec. |
+|volume_write_latency_us | Gauge | `pv_name`=<pv_name> | Integer | Total write latency for all IOs through vol target in usec. |
+
+:::note
+If you require IOPS, Latency, and Throughput in the dashboard, use the following consideration while creating dashboard json config.
+:::
+
+## R/W IOPS Calculation
+
+`num_read_ops` and `num_write_ops` for all resources in stats response are available.
+
+```
+write_iops = num_write_ops (current poll) - num_write_ops (previous_poll) / poll period (in sec)
+```
+
+```
+read_iops = num_read_ops (current poll) - num_read_ops (previous_poll) / poll period (in sec)
+```
+
+## R/W Latency Calculation
+
+`write_latency` (sum of all IO's read_latency) and `read_latency` (sum of all IO and read_latency) are available.
+
+```
+read_latency_avg = read_latency (current poll) - read_latency (previous poll) / num_read_ops (current poll) - num_read_ops (previous_poll)
+```
+
+```
+write_latency_avg = write_latency (current poll) - write_latency (previous poll) / num_write_ops (current poll) - num_write_ops (previous_poll)
+```
+
+## R/W Throughput Calculation
+
+`bytes_read/written` (total bytes read/written for a bdev) are available.
+
+```
+read_throughput = bytes_read (current poll) - bytes_read (previous_poll) / poll period (in sec)
+```
+
+```
+write_throughput = bytes_written (current poll) - bytes_written (previous_poll) / poll period (in sec)
+```
+
+### Handling Counter Reset
+
+If a particular IO engine restarts and when the IO engine comes back online, all the bdev stats will be reset. Users will receive lesser values for all the resources residing on that particular IO engine due to reset. So using the above logic would yield negative values. Hence, the counter current poll is less than the counter previous poll. In this case, do the following:
```

Comment:
The performance stats are not persistent across IO engine restart, this means the counters will be reset upon IO engine restart.